.gitignore

```diff
@@ -26,4 +26,5 @@ adapters/solana/libyellowstone_grpc_geyser.dylib
 
 reveal_*.tx
 
-db
\ No newline at end of file
+db
```

Comment:
This has been a mistake since it would cause the ignore of anything that matches DB... including `sov-db`

Reply:
I have not made any changes regarding the db entry in .gitignore. This line was introduced in a prior commit, which you can refer to [here](https://github.com/chainwayxyz/citrea/commit/22ce56fdf38165d9368374ddb53e890e64decc04).